modules/UI/util/MessageHandler.js

```diff
@@ -476,6 +476,35 @@ var messageHandler = {
     },
 
     /**
+     * Displays a notification.
+     *
+     * @param {string} titleKey - The key from the language file for the title
+     * of the notification.
+     * @param {string} messageKey - The key from the language file for the text
+     * of the message.
+     * @param {Object} messageArguments - The arguments for the message
+     * translation.
+     * @returns {void}
+     */
+    notify: function(titleKey, messageKey, messageArguments) {
+
+        // If we're in ringing state we skip all toaster notifications.
+        if(!notificationsEnabled || APP.UI.isOverlayVisible())
+            return;
+
+        const options = messageArguments
+            ? `data-i18n-options='${JSON.stringify(messageArguments)}'` : "";
+        let element = toastr.info(`
+            <span class="title" data-i18n="${titleKey}" />
+            <br>
+            <span data-i18n="${messageKey}" ${options} />
```

Comment:
Are you certain these self closing tags are okay to use? void elements are common in jsx for elements that don't have content but there's only a handful for html.